src/main/resources/assets/admin/common/js/dom/Element.ts

```diff
@@ -1336,9 +1339,12 @@ export class Element {
     }
 
     private unregisterChildElement(child: Element): number {
```

Comment:
Now sense in returning index now

Reply:
Returned index is used [here](https://github.com/enonic/lib-admin-ui/blob/issue-3066/src/main/resources/assets/admin/common/js/dom/Element.ts#L694-L697).